packages/integration-tests/suites/tracing/browsertracing/meta/template.html

```diff
@@ -2,5 +2,6 @@
   <head>
     <meta charset="utf-8" />
     <meta name="sentry-trace" content="12312012123120121231201212312012-1121201211212012-1" />
+    <meta name="baggage" content="sentry-version=2.1.12" />
```

Comment:
Just wondering if we should prefix the name... we can probably leave it like this - collision is unlikely anyways.